rviz_default_plugins/src/rviz_default_plugins/displays/map/swatch.cpp

```diff
@@ -30,7 +30,6 @@
 
 #include "rviz_default_plugins/displays/map/swatch.hpp"
 
-#include <algorithm>
```

Comment:
I think this should stay included since `std::copy` is used below.

Reply:
True and fixed. Linters are green for me now.